code/datums/mind.dm

```diff
@@ -496,10 +498,14 @@
 		A.admin_remove(usr)
 
 	if (href_list["role_edit"])
-		var/new_role = input("Select new role", "Assigned role", assigned_role) as null|anything in sortList(SSjob.station_jobs)
-		if (!new_role)
+		var/new_role = input("Select new role", "Assigned role", assigned_role.title) as null|anything in sortList(SSjob.station_jobs)
```

Comment:
Not a big priority but there should probably be a way for admins to create custom jobs at runtime for events and such.

Reply:
That is part of my goal, yes. On TGMC I did way more than this, but through several PRs.